R/get_source_expressions.R

```diff
@@ -117,6 +100,30 @@ get_source_expressions <- function(filename) {
   list(expressions = expressions, error = e, lines = source_file$lines)
 }
 
+get_single_source_expression <- function(
```

Comment:
Generally I think we should follow the [tidyverse style guide](https://style.tidyverse.org/functions.html#long-lines-1) for this, which would be either

```r
get_single_source_expression <- function(loc, parsed_content, source_file, filename, tree) {
```

Or

```r
get_single_source_expression <- function(loc,
                                         parsed_content,
                                         source_file,
                                         filename,
                                         tree) {
```